docs/src/installation.md

```diff
@@ -8,6 +8,28 @@ Pkg.add("RCall")
 
 ## Customizing the R installation
 
+There are two ways to configure the R installation used by RCall.jl:
+
+  * [Using Julia's Preferences system](#Customizing-the-R-installation-using-Julia's-Preferences-system)
+  * [At install time, using the `R_HOME` environment variable](#Customizing-the-R-installation-at-install-time)
+
+Should you experience problems with any of these methods, please [open an issue](https://github.com/JuliaStats/RCall.jl/issues/new).
+
+### Customizing the R installation using Julia's Preferences system
+
+In this case the appropriate paths are provided with RCall's `Rhome` and `libR` preferences. You can change these in different ways using [Julia's Preferences system](https://docs.julialang.org/en/v1/manual/code-loading/#preferences). For example, you can add the following to a `LocalPreferences.toml` file in the same directory as a project's `Project.toml` file:
+
+```toml
+[RCall]
+Rhome = "/path/to/env/lib/R"
+libR = "/path/to/env/lib/R/lib/libR.so"
+```
+
+!!! note
+    When these preferences are set, they take precedence over the R installation configured at installation time.
+
```

Comment:
It might be good to add a note here with the method used in the installation tests as a way to use CondaPkg-packaged R as the preferred route.

Reply:
Good idea. Something better should be done here at some point, but I think a "cookbook"-style note is probably best for the meantime.